### ISSUE TITLE
Feature Request: Allow for separate transparency maps when exporting from Blender or using Material Entties.

### ISSUE THREAD

**Issue by Aitolda** [vircadia/vircadia#954]
Right now a PNG is required if transparency is desired, unfortunately this means a MUCH larger file when very often all we need is a 2 color (black and white) jpg for our alpha channel.  It's often the difference between a 5-10mb png vs a sub 1mb jpg.

**Comment by SilverfishVR** [comment]
Material entities already have a "opacityMap" property but as far as I can tell it is not used.
It would be nice to be able to use a seperate opacity map, when importing premade models and avatars that use a seperate texture it is not that easy to merge that into the albedo map.

**Comment by Aitolda** [comment]
> Material entities already have a "opacityMap" property but as far as I can tell it is not used.
> It would be nice to be able to use a seperate opacity map, when importing premade models and avatars that use a seperate texture it is not that easy to merge that into the albedo map.

Yup. I've tried to use the opacityMap property without any luck.

**Comment by daleglass** [comment]
Dev meeting: this is a problem in regards from importing avatars from other systems

**Comment by Aitolda** [comment]
I feel this is still worth considering for people coming in with existing artwork from other platforms.  Most of the MMD's I've seen come through have separate opacity maps.  Sometimes inconveniences like this are part of what hinder newcomers.
